code/datums/gamemode/factions/bloodcult/bloodcult_flavourlines.dm

```diff
@@ -0,0 +1,356 @@
+var/list/failure_lines_by_dept = list(
+	COMMAND_POSITIONS = list(
+		"You have failed to lead them. You would have failed to follow." = 3,
+		"And so ends your 'authority.'" = 3,
+		"This is what passes as command these days." = 3,
+	),
+	ENGINEERING_POSITIONS = list(
+		"Our craft is more complicated than your pathetic tinkering." = 3,
+		"These machines were beyond your skill anyway." = 3,
+	),
+	MEDICAL_POSITIONS = list(
+		"Your refusal helps no one. The blood will still flow." = 3,
+		"There is no cure for this." = 3,
+		"Nothing can heal the veil." = 3,
+	),
+	SCIENCE_POSITIONS = list(
+		"Your closed mind dishonours you." = 3,
+		"Our secrets were beyond your understanding." = 3,
+		"My science was not for wimps such as you." = 3,
```

Comment:
'Runts' or 'weaklings' as opposed to 'wimps', maybe? Chad Nar-Sie bullying nerds is fine, too, though.